### ISSUE TITLE
oscillating accuracy

### ISSUE THREAD

**Issue by kiosklabs** [kiosklabs/kaggle-digit-recognizer-competition#6]
I've noticed that at 15+ epochs, the validation accuracy of our neural network oscillate between 0.95+ and 0.96 valid accuracy. I'm suspecting that this is because of our update_learning_rate value. Pls. confirm if this is correct.

**Comment by kiosklabs** [comment]
update: It seems that running the code over the console **dramatically** improves validation accuracy

# **IPython notebook results:**

Neural Network with 160362 learnable parameters
Layer information
  N  name      size
  0  input     1x28x28
  1  conv1     32x24x24
  2  pool1     32x12x12
  3  conv2     32x8x8
  4  pool2     32x4x4
  5  dense     256
  6  dropout2  256
  7  output    10

  epoch    trn loss    val loss    trn/val    valid acc  dur
      1     2.39802     0.36396    6.58869      0.89233  194.38s
      2     0.42242     0.21683    1.94816      0.93135  206.07s
      3     0.29762     0.16559    1.79728      0.94729  203.16s
      4     0.24954     0.15434    1.61687      0.94955  212.40s
      5     0.22071     0.13913    1.58635      0.95467  205.69s
      6     0.19884     0.12322    1.61367      0.96062  200.09s
      7     0.18463     0.12016    1.53650      0.95979  205.61s
      8     0.17093     0.11670    1.46467      0.96252  208.19s
      9     0.15950     0.11063    1.44174      0.96395  197.47s
     10     0.15099     0.10684    1.41327      **0.96538**  204.32s

# **Terminal results:**

Neural Network with 160362 learnable parameters
Layer information

  N  name      size
  0  input     1x28x28
  1  conv1     32x24x24
  2  pool1     32x12x12
  3  conv2     32x8x8
  4  pool2     32x4x4
  5  dense     256
  6  dropout2  256
  7  output    10

  epoch    trn loss    val loss    trn/val    valid acc  dur
      1     1.68043     0.24406    6.88537      0.92766  218.86s
      2     0.31289     0.16339    1.91498      0.95193  207.28s
      3     0.22779     0.13167    1.72999      0.95681  201.06s
      4     0.18975     0.12192    1.55639      0.96336  220.10s
      5     0.16299     0.10660    1.52898      0.96609  208.96s
      6     0.14583     0.09404    1.55079      0.96954  215.53s
      7     0.12945     0.08540    1.51577      0.97311  200.36s
      8     0.12007     0.08255    1.45443      0.97454  202.66s
      9     0.10815     0.08055    1.34256      0.97537  201.72s
     10     0.10265     0.07745    1.32546      **0.97632**  208.73s

**Comment by kiosklabs** [comment]
@sctcooper let's use kaggle instead for running the tests. they're actually fairly fast, and this side steps your graphic driver issues.
